src/Joseki.jl

```diff
@@ -36,7 +36,11 @@ function stack(fns::Array{Function, 1}, endpoint::Function;
     end
     # Once we have this function we wrap it in a HandlerFunction which is used
     # to register routes
-    return HTTP.HandlerFunction(s)
+    return HTTP.Handlers.HandlerFunction(s)
+end
+
+function register!(router, method::String, url::String, handler)
+    eval(quote HTTP.@register($router, $method, $url, $handler) end)
```

Comment:
There was some discussion about removing the need for this in HTTP in a future release, but this looks good for now.

Reply:
Good to know! And maybe there is already a more elegant way to deal with this?